sdk/identity/test-resources-post.ps1

```diff
@@ -1,11 +1,11 @@
 param (
-    [hashtable] $DeploymentOutputs
+  [hashtable] $DeploymentOutputs
 )
```

Comment:
I would add the below declarations so that calls to `az` throw an error (otherwise they will be silently ignored)

```
$ErrorActionPreference = 'Stop'
$PSNativeCommandUseErrorActionPreference = $true
```